### PR TITLE
[DUOS-2554] Edit Match Endpoint to Return V3 Result

### PR DESCRIPTION
**Pull request by kaitlynoflaherty**
### Addresses
https://broadworkbench.atlassian.net/jira/software/c/projects/DUOS/boards/123?modal=detail&selectedIssue=DUOS-2554

### Summary

----
Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/main/resources/assets/api-docs.yaml [hunk 1]
```diff
@@ -1550,6 +1550,9 @@ components:
         match:
           type: boolean
           description: True if the entities match, false otherwise.
+        abstain:
+          type: boolean
+          description: Nullable boolean will return true if the decision is abstain
```
Comment (rushtong) [inline src/main/resources/assets/api-docs.yaml:1555]: Interesting that we need this in two places, we should probably combine. That can be a separate PR though.

**Review by rushtong — APPROVED** [review]
👍🏽

File: src/main/resources/assets/api-docs.yaml (hunk 1 above)
Comment (kaitlynoflaherty) [reply src/main/resources/assets/api-docs.yaml:1555]: Yeah, I can make a quick ticket for that as well!

**Review by fboulnois — APPROVED** [review]
lgtm
